toolchain/semantics/semantics_ir.h

```diff
@@ -82,6 +82,12 @@ class SemanticsIR {
   }
   auto Print(llvm::raw_ostream& out, bool include_builtins) const -> void;
 
+  // Returns array bound value from the bound node.
+  auto GetArrayBound(SemanticsNodeId bound_id) const -> uint64_t {
```

Comment:
Maybe rename this to `GetArrayBoundValue`? Given the name of this function, I would expect it to take a description of the array type and return the array bound, not take a description of the array bound and return its value.